src/popup/pages/AddToken/styles.tsx

```diff
@@ -0,0 +1,73 @@
+import { StyleRulesCallback, Theme } from '@material-ui/core';
+
+const styles: StyleRulesCallback = (theme: Theme) => ({
+  root: {
+    width: '100%',
+    height: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  contentContainer: {
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    margin: theme.padding.md,
+  },
+  fieldsContainer: {
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  fieldHeading: {
+    marginBottom: theme.padding.unit,
+    fontSize: theme.font.sm,
+    fontWeight: 'bold',
+  },
+  fieldContainer: {
+    marginBottom: theme.padding.custom(8),
+  },
+  detailContainer: {
+    width: '100%',
```

Comment:
can this be handled with `flex: 1`? I remember running into weird bugs if we set the width in other places other than the root of the component.